packages/events/src/index.test.ts

```diff
@@ -28,21 +28,80 @@ afterEach(async () => {
   resetServer()
 })
 
-test('creating a declaration is an idempotent operation', async () => {
+function createClient() {
   const createCaller = createCallerFactory(appRouter)
-
   const caller = createCaller({})
+  return caller
+}
+
+const client = createClient()
+test('event can be created and fetched', async () => {
+  const event = await client.event.create({
+    transactionId: '1',
+    event: { type: 'birth' }
+  })
+
+  const fetchedEvent = await client.event.get(event.id)
+
+  expect(fetchedEvent).toEqual(event)
+})
+
+test('creating an event is an idempotent operation', async () => {
   const db = await getClient()
 
-  await caller.event.create({
+  await client.event.create({
     transactionId: '1',
-    record: { type: 'birth', fields: [] }
+    event: { type: 'birth' }
   })
 
-  await caller.event.create({
+  await client.event.create({
     transactionId: '1',
-    record: { type: 'birth', fields: [] }
+    event: { type: 'birth' }
   })
 
   expect(await db.collection('events').find().toArray()).toHaveLength(1)
 })
+
+test('stored events can be modified', async () => {
+  const originalEvent = await client.event.create({
+    transactionId: '1',
```

Comment:
We could start early by having a seeder mechanism from the beginning

Maybe something like this: 

```
const eventTestSeeder = () => {
  // there are more sophisticated options, of course.
  let i = 0
  const createEventTransaction = (event: { event?: Event }) => {
    return {
      transactionId: i++,
      event: event ?? { type: 'birth' }
    }
  }

  return {
    createEventTransaction
  }
}
```